dendrite-config.yaml

```diff
@@ -18,6 +18,9 @@ matrix:
       - vector.im
       - matrix.org
 
+    # Setting this to false will allow guest user to register on the server
+    guest_access_disabled: true
```

Comment:
We usually name config options from the action they manage, in this case it would rather be `disable_guest_access` (and `DisableGuestAccess` in `config.go` and `register.go`).